gomock/call.go

```diff
@@ -41,11 +41,32 @@ type Call struct {
 	setArgs map[int]reflect.Value
 }
 
+// AnyTimes allows the expectation to be called 0 or more times
 func (c *Call) AnyTimes() *Call {
 	c.minCalls, c.maxCalls = 0, 1e8 // close enough to infinity
 	return c
 }
 
+// MinTimes sets the minimum number of times an expectation must be called. If maxCalls is at the default of 1
```

Comment:
Don't refer to unexported details in an exported method's doc comment. A reader of the API isn't going to understand this.

Try something like this:
```go
// MinTimes requires the call to occur at least n times.
// If AnyTimes and MaxTimes have not been called,
// MinTimes also sets the maximum number of calls to infinity.
```